### ISSUE TITLE
Beschränkung paging beim neuen SRU client

### ISSUE THREAD

**Issue by guenterh** [swissbib/searchconf#49]

**Comment by liowalter** [comment]
Maybe this can help : https://lucene.apache.org/solr/guide/7_4/pagination-of-results.html

**Comment by guenterh** [comment]
Thanks Lionel, this is exactly what I have in mind cursers. I used this long time ago already for Elasticsearch and it's relatively new for SOLR.

But there are some constraints/things to be thought about (which come quickly to my mind, more is possible)
- using such a mechanism isn't sru compatible (I think which is reasonable)
- do I implement this as part of the old client (still version 4.x) or based on new SOLR7 client. My preference is the last one (and I think in V4 this is not possible). But then I would like to finish  this brach https://github.com/swissbib/sruServer/tree/standardCompliantNamespaces with two mayor aims
- make it possible that standard clients like catmandu can use the service (actually not possible because of the namespaces)
- give the whole component "a little bit more love" - refactoring (clean up) and use for example Java 8 features (streams)

I have to think about this and make a balance between our aims.

But: thanks to support thinking about it. I like this a lot, so far I have nobody... Jonas started some days ago because he used the interface for his last project (Elias related)

I used the last SOLR 7 client already here (https://github.com/swissbib/indexerSolrClient) There are some new features which would be nice if you are interested in (I guess you are...)
